.travis.yml

```diff
@@ -21,6 +21,7 @@ env:
   - SCALA_VERSION=2.11 TEST_TYPE=wdlGen
   global:
   #gradle needs this
+  - NO_GCE_CHECK=true
```

Comment:
This is irrelevant but we might as well put it in.  Hmn, I should move it above the comment though...